File: eform-client/src/app/common/models/eforms/template-request.model.ts
export class TemplateRequestModel {
  public sort: string;
  nameFilter: string;
  tagIds: Array<number> = [];
  pageIndex: number;
  pageSize: number;
  isSortDsc: boolean;
  offset: number;

  constructor() {
    this.tagIds = [];
    this.sort = 'id';
    this.nameFilter = '';
    this.isSortDsc = true;
    this.pageSize = 10000;
    this.pageIndex = 0;
    this.offset = 0;
  }
}

File: eFormAPI/eFormAPI.Web/Infrastructure/Models/Templates/TemplateRequestModel.cs
using System.Collections.Generic;

namespace Microting.eFormApi.BasePn.Infrastructure.Models.Templates
{
    public class TemplateRequestModel
    {
        public string Sort { get; set; }
        public string NameFilter { get; set; }
        public int PageIndex { get; set; }
        public int PageSize { get; set; }
        public bool IsSortDsc { get; set; }
        public int Offset { get; set; }
        public List<int> TagIds { get; set; }

        public TemplateRequestModel()
        {
            Sort = "";
            NameFilter = "";
            PageSize = 10;
            PageIndex = 0;
            Offset = 0;
            TagIds = new List<int>();
        }
    }
}